scripts/actions/check-job-progress.js

```diff
@@ -175,14 +176,15 @@ const updateTranslationRecords = async (slugStatuses) => {
 
   await Promise.all(
     slugStatuses.map(async ({ slug }) => {
-      const records = await updateTranslations(
-        { slug, status: StatusEnum.IN_PROGRESS },
-        { status: StatusEnum.COMPLETED }
-      );
-
-      console.log(
-        `Translation ${records[0].id} marked as ${StatusEnum.COMPLETED}`
-      );
+      Object.values(LOCALE_IDS).forEach(async (localeId) => {
```

Comment:
i think we want to have locale be a property on the incoming slugStatus rather than iterate over every locale for every slug.

```
slugStatuses.map(async ({ locale,  slug }) => { . . . }
```

Reply:
i think adding locale [here](https://github.com/newrelic/docs-website/blob/1277a589f91cd4e109cbd0fa84110dc2f9820b21/scripts/actions/check-job-progress.js#L253) would allow us to deconstruct it in this method.